source/common/http/conn_manager_impl.h

```diff
@@ -409,6 +409,8 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     // is ever called, this is set to true so commonContinue resumes processing the 100-Continue.
     bool has_continue_headers_{};
     bool is_head_request_{false};
+    bool decoding_headers_only_{};
```

Comment:
nit: Can you normalize initialization of `is_head_request_` above. Either make them all `{false}` or all empty braces? Don't care which. (As long as you are here.)

Reply:
Done. Maybe it would be possible to add a linter check to check that we're being consistent with initializing members to the default value?